generators/app/templates/generators/client/_index.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-disable consistent-return */
-const chalk = require("chalk");
-const ClientGenerator = require("generator-jhipster/generators/client");
-const prompts = require("./prompts");
-const writeFiles = require("./files").writeFiles;
+const chalk = require('chalk');
```

Comment:
while the applied formatting is fine, we should not run prettier on the actual template files and they should be excluded in prettierignore. Instead we should setup prettier transform on the templates while generating them like in jhipster

Reply:
Alright, that sounds indeed a better idea in the long term :)
I'll check how to do that. I guess something like that is done in generator-jhipster.

Reply:
yes its already done that way in JHipster, look for the prettier setup of generator, generated project and `generator-transform.js`